static-site/components/focus-paragraph/styles.module.css

```diff
@@ -0,0 +1,22 @@
+/* this is the basic style for all FocusParagraph */
+.focus {
+  font-size: var(--niceFontSize);
+  font-weight: 300;
```

Comment:
```suggestion
```

Remove this, and let it inherit from https://github.com/nextstrain/nextstrain.org/blob/a331d36fd8ea14a51a7f4b2fac0926bbc2d41899/static-site/app/styles/globals.css#L28-L34